CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.48.0
+* Enhancement - Docker stop timeout buffer increased from 30s to 2m [2697](https://github.com/aws/amazon-ecs-agent/pull/2697)
+* Enhancement - More informative ENI attachment logs [2703](https://github.com/aws/amazon-ecs-agent/pull/2703)
+* Enhancement - The environment variable ECS_PULL_DEPENDENT_CONTAINERS_UPFRONT=false as default [2731](https://github.com/aws/amazon-ecs-agent/pull/2731) 
```

Comment:
Introduce new environment variable `ECS_PULL_DEPENDENT_CONTAINERS_UPFRONT` to pull images of dependent containers even before `dependsOn` condition is satisfied. This feature is turned off by default.